nixos/modules/programs/nh.nix

```diff
@@ -15,7 +15,7 @@ in
     package = lib.mkPackageOption pkgs "nh" { };
 
     flake = lib.mkOption {
-      type = lib.types.nullOr lib.types.path;
+      type = with lib.types; nullOr (either path str);
```

Comment:
Please correct me if this is not a right way to write such type. I am still pretty new to Nix

Reply:
Actually the `path` type already permits strings:
https://github.com/NixOS/nixpkgs/blob/2cf972b6039f6cec292b018c434719ff73bddbd2/lib/types.nix#L535